ingester/src/stream_handler/handler.rs

```diff
@@ -393,8 +396,30 @@ something clever.",
                         ?op_sequence_number,
                         "successfully applied dml operation"
                     );
+                    // we only want to report the TTBR if anything was applied
+                    if let Some(delta) = duration_since_production {
+                        // Update the TTBR metric before potentially sleeping.
+                        self.time_to_be_readable.set(delta);
+                        trace!(
+                            kafka_topic=%self.topic_name,
+                            shard_index=%self.shard_index,
+                            shard_id=%self.shard_id,
+                            delta=%delta.as_millis(),
+                            "reporting TTBR for shard (ms)"
+                        );
+                    }
                     should_pause
                 }
+                Ok(DmlApplyAction::Skipped) => {
+                    trace!(
+                        kafka_topic=%self.topic_name,
+                        shard_index=%self.shard_index,
+                        false,
+                        ?op_sequence_number,
+                        "did not apply dml operation (skipped)"
```

Comment:
`did not apply dml operation` and `skipped` kinda say the same things - how about we log _why_ it wasn't applied:

```suggestion
                        "did not apply dml operation (op was already persisted previously)"
```

Might help someone reading the logs and being "wtf why not apply my op".